course_contents/1_intro/notes/07. Membership Operators.md

```diff
@@ -0,0 +1,17 @@
+## IN
+- Returns True if an element exists inside an object
+
+Example:
+```python
+users = ['James', 'Charlie', 'Ana']
+print("Johnny" in users)
+```
+
+## NOT IN
+- Returns True if an element does not exist inside an object
+
+Example:
+```python
+users = ['James', 'Charlie', 'Ana']
+print("Johnny" in users)
```

Comment:
Should show the expected output as a comment after the print statements in both code blocks.

Reply:
Done